src/Meshes/mesh.ts

```diff
@@ -1436,6 +1436,20 @@ export class Mesh extends AbstractMesh implements IGetSetVerticesData {
         return this;
     }
 
+    /**
+     * Creates a un-shared specific occurence of the geometry for the mesh.
+     * @returns the current mesh
+     */
+    public makeGeometrySomewhatUnique(): Mesh {

```

Comment:
I'm not a big fan of that naming.
What about CloneGeometry instead?

Reply:
Yeah I left it with a bad name to discuss it if we get that far. What it does is that it clones the geometry instead of copying it... and you can can change vertex buffers if you want to, forgetting the old ones.. but they are there until you do... If we do sort out the index buffer then cloneGeometry might be a good name,